src/components/Disclosures/DisclosureControllable.tsx

```diff
@@ -66,7 +66,9 @@ const DisclosureControllable: FC<DisclosureControllableProps> = ({
           alt=""
         />
       </div>
-      {isOpen && <div id={disclosureId}>{children}</div>}
```

Comment:
I think we should stick to "conditional rendering" instead of class toggling if possible.
No need to render stuff if we can avoid it (explained in cave man language).